src/base/_forms.css

```diff
@@ -53,16 +53,16 @@ details > summary,
 }
 
 .outline {
-  background: transparent;
+  background: transparent !important;
```

Comment:
Is there a way to avoid the !importants on this? Include `.button.outline` maybe?